README.md

```diff
@@ -167,7 +167,7 @@ Installation
 
 ### Standard
 
-_Sibyl_ requires at least _Miasm2_ version `v0.1.1` and the corresponding version of _Elfesteem_.
+_Sibyl_ requires at least _Miasm_ version `v0.1.3` and the corresponding version of _Elfesteem_.
```

Comment:
```suggestion
_Sibyl_ requires at least _Miasm_ version `v0.1.3`.
```

Reply:
Fixed.
I forgot to remove the installation of elfesteem as well, and the use of elfesteem in the code instead of miasm loader